src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -56,6 +56,86 @@
                                        ["code", "lib", "func", "reason"])
 
 
+def _encode_asn1_int(backend, x):
+    """
+    Converts a python integer to a ASN1_INTEGER. The returned ASN1_INTEGER will
+    not be garbage collected (to support adding them to structs that take
+    ownership of the object). Be sure to register it for GC if it will be
+    discarded after use.
+
+    """
+    i = backend._lib.ASN1_INTEGER_new()
+    backend._lib.ASN1_INTEGER_set(i, x)
```

Comment:
This call will fail for any integer > native word size. We should probably convert the python int to BN and then do a conversion of BN to ASN1_INTEGER (I assume a function already exists for this purpose) so we can safely use this as a general purpose converter.